firedantic/models.py

```diff
@@ -1,14 +1,28 @@
 from abc import ABC
-from typing import List, Optional, Type, TypeVar
+from typing import Any, List, Optional, Type, TypeVar
 
 import pydantic
-from google.cloud.firestore_v1 import CollectionReference, DocumentReference
+from google.cloud.firestore_v1 import CollectionReference, DocumentReference, Query
 
 from firedantic.configurations import CONFIGURATIONS
 from firedantic.exceptions import CollectionNotDefined, ModelNotFoundError
 
 TModel = TypeVar("TModel", bound="Model")
 
+# https://firebase.google.com/docs/firestore/query-data/queries#query_operators
+FIND_TYPES = (
```

Comment:
I would prefer a set for this kind of things where you do a lot of membership (something in something) checks.